deploy/manifests/sample-deployments/kong-enterprise-hybrid/kong-cp.env

```diff
@@ -0,0 +1,3 @@
+KONG_ADMIN_API_URI=http://192.168.1.100:30801
+KONG_ADMIN_GUI_URL=http://192.168.1.100:30802
+KONG_ADMIN_GUI_SESSION_CONF={"cookie_secure":false,"storage":"kong","cookie_name":"admin_session","cookie_lifetime":31557600,"cookie_samesite":"off","secret":"the-best-secret"}
```

Comment:
Similar to other comments on sensitive config. This shouldn't provide a usable secret, and should not disable `cookie_secure`. Second half is a bit thornier, but I err on the side of providing examples that target production environments (and have TLS enabled as such).

https://docs.konghq.com/enterprise/2.1.x/kong-manager/authentication/sessions/#configuration-to-use-the-sessions-plugin-with-kong-manager is the standard guidance we provide.

Reply:
I can change this one, i took the same value i found here https://github.com/Kong/kubernetes-ingress-controller/blob/main/deploy/manifests/enterprise/kong-enterprise.yaml (and i think i changed the secret value...)